policyengine_us/parameters/gov/states/ca/tax/income/credits/foster_youth/base.yaml

```diff
@@ -0,0 +1,31 @@
+description: California provides the following foster youth tax credit amount, based on age.  
+brackets:
+  - threshold:  
+      2022-01-01: 0  
+    amount:  
+      2022-01-01: 0  
+  - threshold:  
+      2022-01-01: 18  
+    amount:  
+      2022-01-01: 1_083
```

Comment:
can you check why this isn't $1,176 (amount from the law) * 0.85 (EITC adjustment factor)?

Reply:
Neither the legal code nor the tax form instruction provides an explanation for this.

Reply:
please file an issue to reconcile

Reply:
Filed #3793